server/src/main/java/io/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -42,6 +42,7 @@
 public class RealtimeTuningConfig implements TuningConfig, AppenderatorConfig
 {
   private static final int defaultMaxRowsInMemory = 75000;
+  private static final long defaultMaxBytesInMemory = getDefaultMaxBytesInMemory();
```

Comment:
This should refer to `TuningConfig.DEFAULT_MAX_ROWS_IN_MEMORY`, `TuningConfig.DEFAULT_MAX_BYTES_IN_MEMORY`, or wherever we end up putting it.

Reply:
done